### ISSUE TITLE
Can webagent simulate enter text and click?

### ISSUE THREAD

**Issue by gustavklopp** [huginn/huginn#2657]
Hi!
I need to scrap a web page which is only accessible after using the internal search engine on website homepage.
So I need to select the text box on the homepage, click on the 'Go' button to go the page which I want to scrap.
Is it possible?
Thanks a lot in advance!

**Comment by agaelebe** [comment]
I dont think so. But you probably can do this using PhantomJScloud: https://phantomjscloud.com/docs/http-api/automation/

**Comment by dsander** [comment]
> I dont think so. But you probably can do this using PhantomJScloud: https://phantomjscloud.com/docs/http-api/automation/

Yes PhantomJScloud is a good option if the Website uses Javascript. If the "go" button just triggers a page refresh you can get the URL, the parameters and possibly headers from the developer tools of your browser and use the Website or PostAgent to make the same request with Huginn.
